
Current position
======
__Postdocal researcher__ in evolution and ecology at [CNRS](https://www.cnrs.fr/) and [Université Paris-Saclay](https://www.ideev.universite-paris-saclay.fr/en/) from November 2022.

Working with Tatiana Giraud. 

> sex chromosome evolution in Microbtryum species

__Affiliation__  <br>UMR 8079 CNRS-UPS-AgroParisTech, <br> 12 route 128 91190 Gif-sur-Yvette, France 


Past work 
======


__2020 - 2022__ PostDoc at CNRS CEFE Montpellier with Mathieu Joron. Demography and population genomics of heliconius numata.

__2016 - 2020__  Postdoc at Université Laval with Louis Bernatchez. Modeling demographic and selective process within natural population. 


__2012 - 2015__ Phd Thesis at [INRAE](https://www.umr-decod.fr/en) (French research institute for agronomy), Rennes, France. 
>population genomics and experimental approaches to study speciation in European lampreys. Rennes, France Supervision : Guillaume Evanno, Sophie Launey. 
>
> 4 articles published (see publication list). Phd Thesis [here](https://quentin-rougemont.github.io/doc/ThesisRougemont.pdf)


__2012__ Master thesis at [INRAE](https://www.umr-decod.fr/en) (French research institute for agronomy), Rennes, France. 
> Combining population gentics and otholotith microchemistry to study dispersal and genetic differentiation in Alosa sp. Supervision : Sophie Launey, Elodie Reveillac . 
>
> Article published in 2022 in [Heredity](https://doi.org/10.1038/s41437-022-00547-9), master thesis [here](https://quentin-rougemont.github.io/doc/RapportQR12-09-12.pdf)


Formation 
======

__2017__ Python MOOC online 


__2015__ Approximate Bayesian Computation and Random Forest with Pr. Jérôme Goudet and Camille Roux at [Lausanne University] (https://www.unil.ch/unil/fr/home.html) 

__2009 - 2012__ Master in [Environmental sciences and engineering]() at [l'institut agro](https://www.institut-agro-rennes-angers.fr/), Rennes, France.
> Studied applied ecology and biodiversity, sciences
> Studied population genetics (6 month internship in population genetics of Atlantic salmon at [Exeter University](https://www.exeter.ac.uk/) with [Jamie Stevens](https://experts.exeter.ac.uk/search?by=text&type=user&v=%22jamie%20stevens%22)


Scientific skills 
======

* Genome Assembly 

* Genome Annotation

* Pangenome Graph

* Population genomics and molecular evolution

* Phylogenetics

* Approximate Bayesian Computation - Random Forest


Publication list 
======

**A) Submitted**

36.  Q. Rougemont, E Lucotte, L Boyer, A. Jalaber, A. Snirc, T Giraud, Ricardo C Rodríguez de la Vega, EASYstrata: a new workflow to infer evolutionary strata along sex chromosomes and other supergenes, doi: https://doi.org/10.1101/2025.01.06.631483  

**B) Book Chapter**
35. T Leroy & Q Rougemont (2021) Population genetics analysis methods: from population structure inferences to genome scans for selection. In: Molecular Plant Taxonomy (Springer). https://link.springer.com/protocol/10.1007/978-1-0716-0997-2_16. [here](https://quentinrougemont.github.io/docs/30_Leroy_Rougemont_Chapter_BookMolecularPlantTaxonomy_v2.pdf)


34. H Cayuela, J Prunier, Q Rougemont Genomic approaches to study dispersal in free-ranging animal populations: insights for wildlife management. In applied environnemental genomics. CSIRO. (2023). DOI: 10.1071/9781486314935.[here](https://quentinrougemont.github.io/docs/31_Chapter_22.pdf)

**C) Accepted Articles**

33. L Guichard, R Lagadec, L Michel, H Mayeur, M FUENTES, J Pain, N Heier, Q Rougemont, MC Rodicio, A Barreiro-Iglesias, P Blader, M Schubert and S Mazan, 2024. The lamprey habenula provides an extreme example for the temporal regulation of asymmetric development.  Frontiers in Cell and Developmental Biology, in review

32. C Moraga*, C Branco*, Q Rougemont*, P Veltsos, P Jedlička, [….], D Charlesworth, T Giraud, A. Bendahmane, A. Di Genova, A. Madoui, R Hobza, G.A.B. Marais The Silene latifolia genome and its giant Y chromosome. bioRxiv 2023.09.21.558754; doi: https://doi.org/10.1101/2023.09.21.558754. In review. [here](https://quentinrougemont.github.io/docs/).

31. E Lucotte, P Jay, Q Rougemont, L Boyer, A Cornille, A Snirc, A Labat, E Chahine, M Duhamel, J Gendelman, RK Hayes, MH Perlin, A Widmer, J Antonovics, E Bruns, ME Hood, RC Rodríguez de la Vega, T Giraud Loss of function of HD mating-type genes in anther-smut fungi and recombination suppression without mating-type loci linkage, doi: https://doi.org/10.1101/2024.03.03.583181. In review. [here](https://quentinrougemont.github.io/docs/33_Lucotte_et_al2024_RepeatedLoss.pdf)

30. FE Hartmann, RC Rodriguez de la Vega, Q Rougemont, A Deméné, T Badet, D Croll, A Labat, S Prospero, A Snirc, L Stauber, C Dutech, T Giraud, Million years old recombination suppression in a region linked to the mating-type locus in the chestnut blight fungus Cryphonectria parasitics. doi: https://doi.org/10.1101/2024.03.29.587348  In review.  [here](https://quentinrougemont.github.io/docs/32_Hartman_et_al2024_OldInversionCrypho.pdf)

29. Q Rougemont, T Leroy, EB Rondeau, BF Koop, L Bernatchez (2023) Allele surfing causes maladaptation in a Pacific salmon of conservation concern. Plos Genetics  DOI: 10.1371/journal.pgen.1010918. [here](https://quentinrougemont.github.io/docs/29_Rougemont_et_al_2023_PlosGenetics.pdf)


28. Q Rougemont, B Huber, S Martin, A Whibley, C Estrada, D Solano, R Orpet, W.O McMillan, B Frérot, M Joron (2023) Subtle introgression footprints at the end of the speciation continuum in a clade of Heliconius butterflies. Molecular Biology and Evolution  https://doi.org/10.1093/molbev/msad166 
[here](https://quentinrougemont.github.io/docs/28_Rougemont_et_al2023MBE_Heliconius.pdf)

27. MÁ Rodríguez de Cara*, P Jay*, Q. Rougemont*, M Chouteau, A Whibley, B, F Piron-Prunier, R Rogner Ramos, AVL Freitas, C Salazar, K Lucas Silva-Brandão, T Texeira Torres, M Joron (2023) Supergene formation is associated with a major shift in genome-wide patterns of diversity in a butterfly. Peer-Community Journal (10.24072/pcjournal.298). [here](https://quentinrougemont.github.io/docs/27_DeCara_Jay_Rougemont_2023_pcjournal_298.pdf)


26. EB Rondeau, KA Christensen, DR Minkley, JS Leong, MTT Chan, CA Despins, A Mueller, D Sakhrani, CA Biagi, Q Rougemont, E Normandeau, SJM Jones, RH Devlin, RE Withler, TD Beacham, KA Naish, JM Yáñez, R Neira, L Bernatchez, WS Davidson, BF Koop (2023) Isolation-by-distance and population-size history inferences from the coho salmon (Oncorhynchus kisutch) genome  2023.06.14.496192; G3, https://doi.org/10.1093/g3journal/jkad033 [here](https://quentinrougemont.github.io/docs/26_Rondeau_et_al2022CohoGenome.pdf)


25. A Xuereb, Q Rougemont, JS. More,  E Normandeau, TD Beacham, L Bernatchez (2023) Refining conservation units in Pacific Salmon. Evolutionary applications. https://doi.org/10.1111/eva.13489  [here](https://quentinrougemont.github.io/docs/25_Xuereb_et_al2022Re‐evaluating Coho salmon Oncorhynchus kisutch conservation units in Canada.pdf)

24.Q Rougemont, C Perrier, AL Besnard, I Lebel, Y Abdallah, E Feunteun, E Réveillac, E Lasne, A Acou, DJ Nachon, F Cobo, G Evanno, JL Baglinière, S Launey (2022) Population genetics reveals cryptic lineages and ongoing hybridization in a declining migratory fish species complex Heredity 10.1038/s41437-022-00547-9 [here](https://quentinrougemont.github.io/docs/24_Rougemont_et_al2022AlosaGenetics.pdf)


23. Y Dorant, M Laporte, Q Rougemont, H Cayuela, R Rochette, L Bernatchez (2022). Landscape genomics of the American lobster (Homarus americanus). Molecular Ecology. https://doi.org/10.1111/mec.16653 [here](https://quentinrougemont.github.io/docs/23_Dorant_et_al2022MolEcol_Landscape genomics of the American lobster Homarus americanus.pdf)

22. Q Rougemont, A Xuereb, X Dallaire, JS Moore, E Normandeau, EB Rondeau, RE Withler, DM Van Doornik, PA Crane, KA Naish, JC Garza, TD Beacham, BF Koop, L Bernatchez (2022) Long-distance migration is a major factor driving local adaptation at continental scale in Coho salmon Molecular Ecology https://doi.org/10.1111/mec.16339.
	*  voir perspective: https://doi.org/10.1111/mec.16792 [here](https://quentinrougemont.github.io/docs/22_Rougemont_Molecular_Ecology_2022.pdf)


21. Xuereb A, Rougemont Q, Tiffin P, Xue H, Phifer-Rixey M (2021) Individual-based eco-evolutionary models for understanding adaptation in changing seas. Proc. R. Soc. B https://doi.org/10.1098/rspb.2021.2006   
[here](https://quentinrougemont.github.io/docs/21_xuereb-et-al-2021-individual-based-eco-evolutionary-models-for-understanding-adaptation-in-changing-seas.pdf)

20. L Benestan, Q Rougemont, E Parent, E Normandeau, L Bernatchez, Y Lambert, C Audet, G Parent (2021) Population genomics and history of speciation reveal fishery management gaps in two related redfish species (Sebastes mentella and Sebastes fasciatus). Evolutionary Applications. https://doi.org/10.1111/eva.13143  [here](https://quentinrougemont.github.io/docs/20_Benestan_et_al2020Population genomics and history of speciation reveal fishery management gaps.pdf)


19. Q Rougemont, J-S Moore, T. Leroy, E. Normandeau, E.B. Rondeau, R.E. Withler, D.M.Van Doornik, P.A. Crane, K.A. Naish, J.C Garza, T.D. Beacham, B.F. Koop, L Bernatchez (2020) Demographic history shaped geographical patterns of deleterious mutation load in a broadly distributed Pacific salmon. PLOS Genetics. https://doi.org/10.1371/journal.pgen.1008348 [here](https://quentinrougemont.github.io/docs/19_Rougemont_et_al2020PlosGen.pdf)


18. Q Rougemont, V Dolo, A Oger, AL Besnard, MA Coutellec, S Launey, C Perrier, G Evanno (2020) Riverscape genetics in brook lamprey: genetic diversity is less influenced by river fragmentation than by gene flow with the anadromous ecotype. Heredity. https://doi.org/10.1038/s41437-020-00367-9.  [here](https://quentinrougemont.github.io/docs/18_Rougemont_et_al2020LandscapeGeneticsLamprey.pdf)


17. Q Rougemont*, H Cayuela*, M Laporte, C Mérot, E Normandeau, S Jentoft, P Sirois, M. Castonguay, T Jansen, K Praebel, M Clément, L Bernatchez (2020). Shared ancestral polymorphisms and chromosomal rearrangements as potential drivers of local adaptation in a marine fish. Molecular Ecology. https://doi.org/10.1111/mec.15499  (*: Co-first authorship). [here](https://quentinrougemont.github.io/docs/17_CayuelaRougemont_et_al2020_Shared ancestral polymorphisms and chromosomal rearrangements as potential drivers of.pdf)


16. C Perrier, Q Rougemont, Charmentier A (2020) Demographic history and genomics of local adaptation in blue tits. Evolutionary Applications. https://doi.org/10.1111/eva.13035 [here](https://quentinrougemont.github.io/docs/16_PerrierRougemontCharmentier2020_Demographic history and genomics of local adaptation in blue tit populations.pdf)


15. Y Dorant, H Cayuela, K Wellband, M Laporte, Q Rougemont, C Mérot, E Normandeau, R Rochette, L Bernatchez (2020) Copy number variants outperform SNPs to reveal genotype-temperature association in a marine species. Molecular Ecology. https://doi.org/10.1111/mec.15565  [here](https://quentinrougemont.github.io/docs/15_Dorant_et_al2020_Copy number variants outperform SNPs to reveal genotype temperature association in a.pdf)
	* see perspective: https://doi.org/10.1111/mec.15665 

14. Q Rougemont*,  M Letwein*, M Duranton*, PA Gagnaire, L Bernatchez (2020) Using haplotype information for conservation genomics. Trends in Ecology & Evolution, https://doi.org/10.1016/j.tree.2019.10.012  (*: Co-first authorship). [here](https://quentinrougemont.github.io/docs/14_RougemontLeitweinDurantonTREE2020.pdf)


13. Q Rougemont, A Carrier, J leluyer, AL Ferchaud, JM Farrell, D Hatin, PBrodeur, L Bernatchez (2019) Combining population genomics and forward simulations to investigate stocking impacts: A case study of Muskellunge (Esox masquinongy) from the St. Lawrence River basin. Evolutionary Applications,  https://doi.org/10.1111/eva.12765 [here](https://quentinrougemont.github.io/docs/13_Rougemont_et_al_2019EvolApp_Combining population genomics and forward simulations to investigate.pdf)

12. T Leroy, Q Rougemont, JL Dupouey, C Bodenes, C Lalanne, C Belser, K Labadie, G Le Provost, JM Aury, A Kremer, C Plomion (2019) Massive postglacial gene flow between European white oaks uncovered genes underlying species barriers. New Phytologist, https://doi.org/10.1101/246637. [here](https://quentinrougemont.github.io/docs/12_LeroyRougemont_et_al2019NewPhytol_Massive postglacial gene flow between European white oaks uncovered genes underlying.pdf)

11. Y Dorant, Q Rougemont*, L Benestan*, E Normandeau, B Boyle, R Rochette, L Bernatchez (2019) Comparing Pool-seq, Rapture and GBS genotyping for inferring population structure; the American lobster (Homarus americanus) as a case study. Ecology & Evolution, https://doi.org/10.1002/ece3.5240 (*: Co-authorship) [here](https://quentinrougemont.github.io/docs/11_Dorant_et_al2019_Comparing Pool‐seq Rapture and GBS genotyping for inferring weak population.pdf)


10. H Cayuela, Q Rougemont, JG Prunier, JS Moore, J Clobert, A Besnard, L Bernatchez (2018) Demographic and genetic approaches to study dispersal in wild animal populations: a methodological review. Molecular Ecology, https://doi.org/10.1111/mec.14848. [here](https://quentinrougemont.github.io/docs/10_Cayuela_et_al2018_Demographic and genetic approaches to study dispersal in wild animal populations A.pdf)


9. Q Rougemont* L Bernatchez (2018) The demographic history of Atlantic salmon (Salmo salar) across its distribution range reconstructed from approximate Bayesian computations. Evolution, https://doi.org/10.1111/evo.13486 [here](https://quentinrougemont.github.io/docs/09_Rougemont_Bernatchez2018EvolutionSalmoSalar.pdf)

	* voir perspective : https://doi:10.1111/evo.13504 

8. M Voillemot, Q Rougemont, C Roux JR Pannell (2018) The divergence history of the perennial plant Linaria cavanillesii confirms a recent loss of self-incompatibility. Journal of Evolutionary Biology, https://doi.org/10.1111/jeb.13209 [here](https://quentinrougemont.github.io/docs/08_VoilemotRougemont2018SelfIncompatibilityHistory.pdf)


7. JS Moore, LN. Harris, J LeLuyer, BJ Sutherland, Q Rougemont, RF Tallman, AT Fisk L. Bernatchez (2017) Migratory difficulty drives habitat choice and local adaptation in anadromous Arctic Char (Salvelinus alpinus): evidence from integration of genomics and acoustic telemetry. Molecular Ecology, https://doi.org/10.1111/mec.14393  [here](https://quentinrougemont.github.io/docs/07_Moore_et_al2017MolEcol_Genomics and telemetry suggest a role for migration harshness in determining.pdf)


6. Q Rougemont, PA Gagnaire, C Perrier, C Genthon, AL Besnard, S Launey G Evanno (2017) Parallel patterns of genomic divergence among multiple pairs of lamprey ecotypes explained by a common history of divergence and secondary gene flow.  Molecular Ecology, https://doi.org/10.1111/mec.13664 [here](https://quentinrougemont.github.io/docs/06_Rougemont_et_al_2017MolecularEcology.pdf)


5. Q Rougemont, C Roux S Neuenschwander, J Goudet G Evanno (2016) Reconstructing the demograhic history of divergence between European river and brook lampreys using Approximate Bayesian Computations. Peer-j, https://peerj.com/articles/1910/ [here](https://quentinrougemont.github.io/docs/05_Rougemonte_et_al2016PeerJ.pdf)


4. Q Rougemont, A Gaigher, E Lasne, J Côte, M Coke, AL Besnard, S Launey G Evanno (2015) Low reproductive isolation and highly variable levels of gene flow reveal limited progress toward speciation between European river and brook lampreys, Journal of Evolutionary Biology, https://doi.org/10.1111/jeb.12750 [here](https://quentinrougemont.github.io/docs/04_Rougemont_et_al_2105JEB_LampreySpeciation.pdf)


3. J Martin Q Rougemont, H Drouineau, S Launey, P Jattteau, G Bareille, S Berail, C Pécheyran, E Feunteun, S Roques, D Clavé, DJ Nachon, C Antunes, M Mota, E Réveillac F Daverat (2015) Dispersal capacities of anadromous Allis shad population inferred from a coupled genetic and otolith approach. Canadian Journal of Fisheries and Aquatic Science, https://doi.org/10.1139/cjfas-2014-0510. [here](https://quentinrougemont.github.io/docs/03_Martin_et_al_2015_AllisShad.pdf)


2. R Lagadec, L Laguerre, A Menuet, A Amara, C Rocancourt, P Péricard, BG Godard, MC Rodicio, I. Rodriguez-Moldes, H Mayeur, Q Rougemont, S Mazan and A Boutet (2015)  The ancestral role of Nodal signaling in breaking L/R symmetry in the vertebrate forebrain, Nature Communication. https://www.nature.com/articles/ncomms7686 [here](https://quentinrougemont.github.io/docs/01_Rougemont_et_al_2015_AlisShadMicroSat.pdf)


1. Q Rougemont, AL Besnard, JL Baglinière S Launey (2014)  Characterization of thirteen new microsatellite markers for allis shad (Alosa alosa) and twaite shad (Alosa fallax). Conservation Genetic Ressources, https://doi.org/10.1007/s12686-014-0352-z [here](https://quentinrougemont.github.io/docs/02_Lagadec_et_al2015LampreyNodalSignaling.pdf)

